docs/querying/sql-data-types.md

```diff
@@ -67,53 +66,51 @@ The following table describes how Druid maps SQL types onto native types when ru
-|DATE|LONG|`0`, meaning 1970-01-01|Casting TIMESTAMP to DATE rounds down the timestamp to the nearest day. Casts between string and date types assume standard SQL formatting, e.g. `2000-01-02`. For handling other formats, use one of the [time functions](sql-scalar.md#date-and-time-functions).|
-|ARRAY|ARRAY|`NULL`|Druid native array types work as SQL arrays, and multi-value strings can be converted to arrays. See the [`ARRAY` details](#arrays).|
+|TIMESTAMP|LONG|`0`, meaning 1970-01-01 00:00:00 UTC|Druid's `__time` column is reported as TIMESTAMP. Casts between string and timestamp types assume standard SQL formatting&mdash;for example, `2000-01-02 03:04:05`&mdash;not ISO8601 formatting. For handling other formats, use one of the [time functions](sql-scalar.md#date-and-time-functions).|
+|DATE|LONG|`0`, meaning 1970-01-01|Casting TIMESTAMP to DATE rounds down the timestamp to the nearest day. Casts between string and date types assume standard SQL formatting&mdash;for example, `2000-01-02`. For handling other formats, use one of the [time functions](sql-scalar.md#date-and-time-functions).|
+|ARRAY|ARRAY|`NULL`|Druid native array types work as SQL arrays, and multi-value strings can be converted to arrays. See [Arrays](#arrays) for more information.|
 |OTHER|COMPLEX|none|May represent various Druid column types such as hyperUnique, approxHistogram, etc.|
 
 <sup>*</sup> Default value applies if `druid.generic.useDefaultValueForNull = true` (the default mode). Otherwise, the default value is `NULL` for all types.
 
 ## Multi-value strings
 
 Druid's native type system allows strings to potentially have multiple values. These
-[multi-value string dimensions](multi-value-dimensions.md) are reported in SQL as `VARCHAR` typed, and can be
-syntactically used like any other `VARCHAR`. Regular string functions that refer to multi-value string dimensions are
+[multi-value string dimensions](multi-value-dimensions.md) are reported in SQL as VARCHAR typed, and can be
+syntactically used like any other VARCHAR. Regular string functions that refer to multi-value string dimensions are
 applied to all values for each row individually. Multi-value string dimensions can also be treated as arrays via special
 [multi-value string functions](sql-multivalue-string-functions.md), which can perform powerful array-aware operations, but retain
-their `VARCHAR` typing and behavior.
+their VARCHAR typing and behavior.
 
 Grouping by a multi-value expression observes the native Druid multi-value aggregation behavior, which is similar to
-an implicit SQL `UNNEST`. Refer to the documentation on [multi-value string dimensions](multi-value-dimensions.md)
+the implicit SQL UNNEST operator. Refer to the documentation on [multi-value string dimensions](multi-value-dimensions.md)
```

Comment:
Removed the word "implicit".

Reply:
This is how I interpret the various sentences:
* Original: Grouping behaves similarly to an UNNEST operation that happens implicitly
* Change 1: Grouping behaves similarly to UNNEST, which is an implicit operator
* Change 2: Grouping behaves similarly to the UNNEST operator

Not sure if this is splitting hairs but it seems the meaning is slightly different for each.

Reply:
MVD groupby behaves like UNNEST in the sense that it can return more rows in your result than there are in the original table. I think we'd want to include some of the detail that exists on the MVD page. Specifically, this section: https://druid.apache.org/docs/latest/querying/multi-value-dimensions.html#grouping